src/app/containers/ATIAnalytics/beacon/index.js

```diff
@@ -2,26 +2,31 @@ import sendBeacon from '../../../lib/analyticsUtils/sendBeacon';
 import { buildATIEventTrackUrl } from '../atiUrl';
 
 export const sendEventBeacon = async ({
+  campaignName,
   componentName,
+  format,
+  pageIdentifier,
+  platform,
+  service,
+  statsDestination,
   type,
-  componentInfo,
-  ...params
+  url,
 }) => {
-  const eventProps = {
-    ...params,
-    componentName,
-    componentInfo,
-    type,
-  };
+  await sendBeacon(
+    buildATIEventTrackUrl({
+      campaignName,
+      componentName,
+      format,
+      pageIdentifier,
+      platform,
+      service,
+      statsDestination,
+      type,
+      url,
+    }),
+  );
 
-  const viewProps = {
-    ...eventProps,
-    type: 'view',
-  };
-  await Promise.all([
-    sendBeacon(buildATIEventTrackUrl(eventProps)), // event, e.g. click
-    sendBeacon(buildATIEventTrackUrl(viewProps)), // background event
-  ]);
+  return Promise.resolve();
```

Comment:
```suggestion

```
I don't think this line is needed in an async function? I think you only need to use the promise api in situations like above where you are awaiting multiple asyncronous functions in paralell.